pkg/telemetery/settings.go

```diff
@@ -0,0 +1,15 @@
+package telemetery
+
+import (
+	"github.com/jaegertracing/jaeger/pkg/healthcheck"
+	"github.com/jaegertracing/jaeger/pkg/jtracer"
+	"github.com/jaegertracing/jaeger/plugin/metrics"
+	"go.uber.org/zap"
+)
+
+type Setting struct {
+	Logger        *zap.Logger
+	Tracer        *jtracer.JTracer
+	MeterProvider metrics.Factory
+	HC            *healthcheck.HealthCheck
```

Comment:
I prefer we switch directly to OTEL's status reporter
```
ReportStatus func(*StatusEvent)
```

Docs https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/component-status.md

We can create a simple adapter that translates a call to ReportStatus() into our HealthCheck API such that the v1 binaries can still work.

Reply:
Added to To-Do list in the doc.